[0m07:57:22.934447 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000260C48AD7C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000260C48AD730>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000260C48AD8B0>]}


============================== 07:57:22.946682 | b8431928-773f-4057-a3b2-b3601651e9c4 ==============================
[0m07:57:22.946682 [info ] [MainThread]: Running with dbt=1.7.0
[0m07:57:22.947646 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\prasa\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt debug', 'send_anonymous_usage_stats': 'True'}
[0m07:57:22.948646 [info ] [MainThread]: dbt version: 1.7.0
[0m07:57:22.950649 [info ] [MainThread]: python version: 3.12.1
[0m07:57:22.951972 [info ] [MainThread]: python path: c:\home_assignment_v1\dbt_venv\Scripts\python.exe
[0m07:57:22.952982 [info ] [MainThread]: os info: Windows-11-10.0.22621-SP0
[0m07:57:22.952982 [info ] [MainThread]: Using profiles dir at C:\Users\prasa\.dbt
[0m07:57:22.954082 [info ] [MainThread]: Using profiles.yml file at C:\Users\prasa\.dbt\profiles.yml
[0m07:57:22.954990 [info ] [MainThread]: Using dbt_project.yml file at C:\home_assignment_v1\dbt_venv\Scripts\dbt_project.yml
[0m07:57:22.956129 [info ] [MainThread]: Configuration:
[0m07:57:22.957138 [info ] [MainThread]:   profiles.yml file [[31mERROR not found[0m]
[0m07:57:22.957903 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m07:57:22.958910 [info ] [MainThread]: Required dependencies:
[0m07:57:22.959920 [debug] [MainThread]: Executing "git --help"
[0m07:57:22.974503 [info ] [MainThread]:  - git [[31mERROR[0m]

[0m07:57:22.976502 [info ] [MainThread]: Connection test skipped since no profile was found
[0m07:57:22.978502 [info ] [MainThread]: [31m3 checks failed:[0m
[0m07:57:22.980505 [info ] [MainThread]: dbt looked for a profiles.yml file in C:\Users\prasa\.dbt\profiles.yml, but did
not find one. For more information on configuring your profile, consult the
documentation:

https://docs.getdbt.com/docs/configure-your-profile


[0m07:57:22.983504 [info ] [MainThread]: Project loading failed for the following reason:
 project path <C:\home_assignment_v1\dbt_venv\Scripts\dbt_project.yml> not found

[0m07:57:22.987507 [info ] [MainThread]: Error from git --help: Could not find command, ensure it is in the user's PATH and that the user has permissions to run it: "git"

[0m07:57:22.991502 [debug] [MainThread]: Command `dbt debug` failed at 07:57:22.991502 after 0.10 seconds
[0m07:57:22.992503 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000260C4777B30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000260C472B7A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000260C3F67FB0>]}
[0m07:57:22.993502 [debug] [MainThread]: Flushing usage events
